libraries/AP_Landing/AP_Landing_Slope.cpp

```diff
@@ -405,14 +407,15 @@ void AP_Landing::type_slope_log(void) const
 // @Field: slope: Slope to landing point
 // @Field: slopeInit: Initial slope to landing point
 // @Field: altO: Rangefinder correction
-    AP::logger().Write("LAND", "TimeUS,stage,f1,f2,slope,slopeInit,altO", "QBBBfff",
```

Comment:
height_flare_log field documentation is missing

Reply:
Thanks, fixed. Now only the new test fails as expected.